docs/faq.md

```diff
@@ -0,0 +1,103 @@
+# Frequently asked questions
+
+### What are those emojis and what do they mean?
+
+Those emojis indicate what the bot does with a message:  
+
+- 👀 ➜ Found a thankword
+- 💤 ➜ Receiver was on cooldown
+- 🔍 ➜ No receiver was found
+- ❓ ➜ Receiver has no recent messages in this channel and is mising in the context
+- ❔ ➜ Donor is not present in channel
+- 🕛 ➜ The referenced message is too old
+- 🗨️ ➜ User was prompted for reputation.
+
+A bot admin can disable this emojis in the `repsettings`.
+
+### Why cant I give reputation to user XY?
+
+This can have several reasons:
```

Comment:
Maybe also link to the abuse protection page which includes more detailed information about this?